code/lib/types/src/modules/addons.ts

```diff
@@ -301,15 +303,19 @@ export interface Addon_RenderOptions {
   key?: string;
 }
 
+/**
+ * @deprecated This type is deprecated and will be removed in 8.0.
+ */
 export type ReactJSXElement = {
   type: any;
   props: any;
   key: any;
 };
 
-export interface Addon_Type {
-  title: (() => string) | string | ReactJSXElement;
-  type?: Addon_Types;
+export type Addon_Type = Addon_BaseType;
+export interface Addon_BaseType {
+  title: FCWithoutChildren | string | ReactElement | ReactNode;
```

Comment:
```suggestion
  title: FCWithoutChildren | ReactNode;
```

`ReactNode` already contains `string` and `ReactElement` (even in Reacty 16+ AFAIK)

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L257-L266

Reply:
I see, it's redundant.